### PR TITLE
Move to RHEL

### PR DESCRIPTION
**Pull request by abs51295**
Depends on https://github.com/openshiftio/openshiftio-cico-jobs/pull/632 to be merged first.

### CODE REVIEW

**Comment by abs51295** [comment]
Don't merge this unless the corresponding cico_jobs PR is merged.

**Comment by abs51295** [comment]
@riuvshin PTAL. I was unable to add you as a reviewer. Thanks :)

File: Dockerfile.rhel [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+COPY ./src /src
+
+RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
+
+ADD scripts/entrypoint.sh /bin/entrypoint.sh
+
+RUN chmod 777 /bin/entrypoint.sh
+
+ENTRYPOINT ["/bin/entrypoint.sh"]
```
Comment (riuvshin) [inline Dockerfile.rhel]: please add new line

File: Makefile [hunk 2]
```diff
@@ -25,5 +31,7 @@ get-image-name:
 	@echo $(REGISTRY)/$(REPOSITORY):$(DEFAULT_TAG)
 
 get-image-repository:
-	@echo $(REPOSITORY)
-
+ 	@echo $(REPOSITORY)
+ 
+get-push-registry:
+	@echo $(REGISTRY)
```
Comment (riuvshin) [inline Makefile]: please add new line

**Pull request merged**